akka-http-core/src/main/scala/akka/http/scaladsl/settings/Http2ServerSettings.scala

```diff
@@ -114,6 +114,9 @@ trait Http2ClientSettings extends /*FIXME: javadsl.settings.Http2ClientSettings
   def incomingStreamLevelBufferSize: Int
   def withIncomingStreamLevelBufferSize(newValue: Int): Http2ClientSettings = copy(incomingStreamLevelBufferSize = newValue)
 
+  def enablePush: Boolean
+  def withEnablePush(newValue: Boolean): Http2ClientSettings = copy(enablePush = newValue)
```

Comment:
This is a client-only setting so not adding it to `Http2CommonSettings`.

Mental note: having peer-specific settings will require refactoring `class Http2ServerDemux(http2Settings: Http2CommonSettings, ... `